listenbrainz/webserver/static/js/src/follow/FollowerFollowingModal.tsx

```diff
@@ -65,24 +73,46 @@ export default class FollowerFollowingModal extends React.Component<
             </Pill>
           </div>
         </div>
-        <div className="follower-following-list">
-          {activeModeList.map((listEntry: string) => {
-            const formattedAsUser: ListenBrainzUser = {
-              name: listEntry,
-            };
-            return (
-              <UserListModalEntry
-                mode="follow-following"
-                key={listEntry}
-                user={formattedAsUser}
-                loggedInUserFollowsUser={loggedInUserFollowsUser(
-                  formattedAsUser
-                )}
-                updateFollowingList={updateFollowingList}
-              />
-            );
-          })}
-        </div>
+
+        {activeModeList.length === 0 ? (
+          <>
+            <hr style={{ margin: "0px 2em", borderTop: "1px solid #eee" }} />
```

Comment:
Maybe better defined in a less file as well. Would it work as a border-top and margins applied to the follower-following-empty class?

Reply:
No because the <hr/> is not a  child of `follower-following-empty` class. I have applied the properties on <hr/> element in follow.less file. It's working.